pkg/ebpf/config.go

```diff
@@ -75,6 +75,10 @@ type Config struct {
 
 	// AttachKprobesWithKprobeEventsABI uses the kprobe_events ABI to attach kprobes rather than the newer perf ABI.
 	AttachKprobesWithKprobeEventsABI bool
+
+	// BypassEnabled is used in tests only.
+	// It enables a ebpf-manager feature to bypass programs on-demand for controlled visibility.
+	BypassEnabled bool
```

Comment:
Moved from `pkg/network/config/Config` to here as it seems the more appropriate place. Also, it needed to be here so that `sharedlibraries.EbpfProgram` depended on the `ebpf.Config` object and not the `network.Config` one.